moveit_ros/perception/mesh_filter/include/moveit/mesh_filter/depth_self_filter_nodelet.h

```diff
@@ -105,10 +105,10 @@ class DepthSelfFiltering : public nodelet::Nodelet
   int queue_size_;
   TransformProvider transform_provider_;
 
-  cv_bridge::CvImagePtr filtered_depth_ptr_;
-  cv_bridge::CvImagePtr filtered_label_ptr_;
-  cv_bridge::CvImagePtr model_depth_ptr_;
-  cv_bridge::CvImagePtr model_label_ptr_;
+  std::shared_ptr<cv_bridge::CvImage> filtered_depth_ptr_;
+  std::shared_ptr<cv_bridge::CvImage> filtered_label_ptr_;
+  std::shared_ptr<cv_bridge::CvImage> model_depth_ptr_;
+  std::shared_ptr<cv_bridge::CvImage> model_label_ptr_;
```

Comment:
This moves from `boost` to `std`. Our own updaters do not set this from any API that relies on the boost pointers, so I'm ok with the change in `master`.

Reply:
Thank you for clarifying the rationale behind allowing the change. Could you elaborate a bit more, because I don't fully understand it. 

- What are the updaters? Are they clean up scripts? 
- Do you mean with the second half, that these pointers are not exposed publically, so they can be changed with this PR? Otherwise, it has to be done with a deprecation?
- Do you mean that this change should only be in `master` not in `noetic-devel`?

Reply:
- We do not differentiate between `master` and `noetic-devel`.
- Ignore the parts about updaters, I thought this change was in a different module when I wrote the comment.
- These are private members of an implementation class, so there is no harm in changing them.